integration-tests/common/common.go

```diff
@@ -1,349 +1,127 @@
 package common
```

Comment:
I didn't bring everything over from starknet for this file. Chances are a decent amount can be brought over mostly as is when writing the test.

Reply:
I wonder if there's more code we can abstract away and share between chains, `common.go`/`test_common.go` etc have quite a lot of code that looks mostly the same in each integration

Reply:
Oh we definitely can. I just wasn't worried about it right now since it will likely all get rewritten again for the BIF work with test templates and common libraries.